docs/questions/query-builder/expressions/converttimezone.md

```diff
@@ -0,0 +1,155 @@
+---
+title: ConvertTimezone
+---
+
+# ConvertTimezone
+
+`convertTimezone` shifts a datetime into a specified time zone by adding or subtracting the right number of hours from the datetime.
+
+Time zones are rather nasty to work with (it's easy to make mistakes, and difficult to catch them), so you should only try to use `convertTimezone` if the interpretation of your data is very sensitive to time-based cutoffs.
+
+For example, if you're tracking user logins over time, you probably won't run your business differently if some logins get counted on Mondays instead of Tuesdays. However, if you're using Metabase to do something precise, like your taxes, you (and the government) will probably care a lot more about the difference between transactions that occurred on Dec 31 vs. Jan 1.
+
+| Syntax                                                               | Example                                                       |
+|----------------------------------------------------------------------|---------------------------------------------------------------|
+| `convertTimezone(column, target, source)`                            | `convertTimezone("December 28, 2022, 12:00:00", "EST, "PST")` |
+| Shifts a datetime from the source time zone to the target time zone. | `December 28, 2022, 9:00:00`                                  |
+
+## Parameters
+
+- `target` is the time zone you want to assign to your column.
+- The name of the `target` time zone depends on your database. For example, you may have to use "Canada/Eastern" instead of "EST".
+- `source` is only a required parameter if you have timestamps without time zones in your database. See [Limitations](#limitations) for more info.
+
+## Creating custom report dates
+
+Let's say that you have some time series data that's stored in one or more time zones (**Source Time**). You want to create custom reporting dates for a team that lives in EST. We recommend labeling `convertTimezone` columns with the name of the target time zone (or adding the time zone to the metadata of a model).
+
+| Source Time                        | Team Report Time (EST)       |
+|------------------------------------|------------------------------|
+| December 28, 2022, 10:00:00        | December 28, 2022, 07:00:00  |
+| December 28, 2022, 21:00:00        | December 28, 2022, 19:00:00  |
+| December 27, 2022, 08:00:00        | December 27, 2022, 05:00:00  |
+
+If you're unsure about the time zone for **Source Time**, you should use two `convertTimezone` expressions to create **Team Report Time (EST)** :
+
+```
+convertTimezone(convertTimezone([Source Time], 'UTC'), 'EST', 'UTC')
```

Comment:
Can we provide an explanatory counterfactual here to explain why someone would do this? That is: what would/could happen if we didn't know the source time zone, and we didn't nest `convertTimezone` expressions to convert the original to UTC?

(Aside: this baselining to UTC seems like something `covertTimezone` should do for you.)

Reply:
Gonna remove this suggestion. I double checked the logic for `convertTimezone` and I think I misremembered this comment from Cal's explanation (should go in the `now` doc instead):

> If you want to do comparisons between a timestamp with time zone and timestamp without time zone, you should convert them to the same time zone first.